xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -143,6 +171,37 @@ public ConfigurationParser(String xml) throws IOException, SAXException, JAXBExc
     this.config = unmarshaller.unmarshal(dom, configTypeClass).getValue();
   }
 
+  private void substituteSystemProperties(final Element dom) {
```

Comment:
Could be static, but I rather not have to deal with two doms... Maintaining a copy just for replacing a few strings (mostly probably not even one!) seems seriously overkill